themes/default/content/blog/pulumi-release-notes-68/index.md

```diff
@@ -0,0 +1,128 @@
+
+## Pulumi CLI and core technologies
+
+### Public preview of Update Plans
+
+Pulumi’s previews are an important part of any workflow where you want to see the changes that will be made to your infrastructure before actually making the changes (with `pulumi up`). However, today there is no guarantee that the `pulumi up` operation will do only what was previewed; if the program, or your infrastructure, changes between the preview and the update, the update might make additional changes to bring your infrastructure back in line with what’s defined in your program. We’ve heard from many of you that you need a strong guarantee about exactly which changes an update will make to your infrastructure, especially in critical and production environments.
+
+As a result, we launched a public preview of Update Plans, a new Pulumi feature which guarantees that operations shown in `pulumi preview` will run on `pulumi up`. Update Plans also help catch any unexpected changes that might happen between when you preview a change and when you apply that change. Update Plans work by saving the results of a `pulumi preview` to a plan file, which enables you to restrict subsequent `pulumi up` operations to only the actions saved in the plan file. This helps you ensure that what you saw in the `pulumi preview` is what will actually happen when you run `pulumi up`.
+
+Learn more in our [blog post on Update Plans]({{< relref "/blog/announcing-public-preview-update-plans" >}}) and in this [Update Plans GitHub issue](https://github.com/pulumi/pulumi/issues/2318).
+
+
+### Disable default provider
+
+While default providers are enabled by default, you can now [disable default providers]({{< relref
+"/docs/intro/concepts/config#special-configuration-options" >}}) on a per stack basis. Disabling default
+providers is a good idea if you want to ensure that your programs must be explicit about which provider they
+will use. For example, to disable the `aws` provider, you can run:
+
+```sh
+$ pulumi config set --path 'pulumi:disable-default-providers[0]' aws
+```
+
+If you wanted to also disable the `kubernetes` default provider, as well as the `aws` default provider, you could run:
+
+```sh
+$ pulumi config set --path 'pulumi:disable-default-providers[1]' kubernetes
+```
+
+This adds a new entry to the list `pulumi:disable-default-providers`. To disable all default providers, use `*` as the package name:
+
+```sh
+$ pulumi config set --path 'pulumi:disable-default-providers[0]' '*'
+```
+Learn more in the [Disable Default Providers GitHub issue](https://github.com/pulumi/pulumi/issues/3383).
+
+### `pulumi up --target` wildcard URN support
+
+Our community had requested that we add wildcard URN support to `pulumi up --target` and as a result we now support wildcards for `pulumi up --target <urn>` and similar commands. Learn more in the [Wildcards support GitHub issue](https://github.com/pulumi/pulumi/issues/5870).
```

Comment:
Might remove the “our community requested” - that’s true for everything here :-). Might instead note that users of the —target flag had requested the ability to use wildcards instead of manually specifying each targeted resource.